### ISSUE TITLE
Getting SyntaxError when using await on version 0.5.0

### ISSUE THREAD

**Issue by kuritz** [actions/github-script#27]
After rolling the version forward to `0.5.0`, my scripts that use `await` are now failing with:

```
SyntaxError: await is only valid in async function
```

My repo is private, so I created an example workflow with the below steps. Full thing is here: https://github.com/kuritz/gtihub-script-test/blob/master/.github/workflows/push.yml
```
      - name: List checks for ref 0.4.0
        uses: actions/github-script@0.4.0
        with:
          github-token: ${{ github.token }}
          script: |
            const result = await github.checks.listForRef({
              owner: context.repo.owner,
              repo: context.repo.repo,
              ref: context.sha,
              status: "in_progress"
            })
            console.log(result)

      - name: List checks for ref 0.5.0
        uses: actions/github-script@0.5.0
        with:
          github-token: ${{ github.token }}
          script: |
            const result = await github.checks.listForRef({
              owner: context.repo.owner,
              repo: context.repo.repo,
              ref: context.sha,
              status: "in_progress"
            })
            console.log(result)
```

And here is a test run of the above: https://github.com/kuritz/gtihub-script-test/runs/462672595?check_suite_focus=true

**Comment by jclem** [comment]
Well that's not right! Looking now.

**Comment by jclem** [comment]
Oof, my bad. I targeted an older version of ECMAScript when compiling the action. I've published [0.6.0](https://github.com/actions/github-script/releases/tag/0.6.0) which should resolve this.

Sorry for the trouble!
